internal/tfimport/importer/google_resource_manager_lien.go

```diff
@@ -30,6 +30,11 @@ type ResourceManagerLien struct{}
 
 // ImportID returns the ID of the resource to use in importing.
 func (i *ResourceManagerLien) ImportID(rc terraform.ResourceChange, pcv ConfigMap, interactive bool) (string, error) {
+	// Can't import if not interactive, since lien names are system-generated and will never be in the plan.
+	if !interactive {
```

Comment:
optional: it seems like a lot of resources will match this criteria so we will need to keep repeating this logic for it. Could we just add a bool field to indicate this can't be imported automatically and then do a check for it once in the base tfimport code?

Reply:
Done, added a common code path for this. However, in the interest of defense-in-depth, since the ImportID function takes the interactive flag, I'd like to leave those "just in case" checks in there too.

Reply:
IMO less code == fewer chances for bugs ;) 

but this is fine

Does it still need the interactive flag if common path handles it?